app/org/sagebionetworks/bridge/play/controllers/BackfillController.java

```diff
@@ -1,50 +0,0 @@
-package org.sagebionetworks.bridge.play.controllers;
-
-import static org.sagebionetworks.bridge.Roles.ADMIN;
-
-import org.sagebionetworks.bridge.models.accounts.UserSession;
-import org.sagebionetworks.bridge.services.backfill.BackfillService;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.beans.BeansException;
-import org.springframework.context.ApplicationContext;
-import org.springframework.context.ApplicationContextAware;
-import org.springframework.stereotype.Controller;
-
-import play.mvc.Result;
-
-@Controller
-public class BackfillController extends BaseController implements ApplicationContextAware  {
```

Comment:
Backfill is used to redrive failed uploads. We can file a separate JIRA to build a new redrive system that doesn't rely on HTTP, but I think for now, we need to keep this.

Reply:
Sorry I didn't know this. I'll restore those files.

Reply:
Restored